app/views/shared/_navbar.html.erb

```diff
@@ -1,4 +1,4 @@
-<nav class="navbar navbar-expand-lg navbar-light bg-light" style="background-color: #e9ecef;">
+<nav class="navbar navbar-expand-lg navbar-light bg-light" style="background-color: #dee2e6;">
```

Comment:
@lime666 we don't write styles on views. Please move this style to the `users.scss` file.